Source/ablastr/mesh_refinement/coarsenToCellCentered.H

```diff
@@ -70,19 +88,18 @@ namespace CoarsenIO{
         const int jmin = idx_min[1];
         const int kmin = idx_min[2];
         int  ii, jj, kk;
-        Real wx, wy, wz;
+        amrex::Real const wx = 1.0_rt / static_cast<amrex::Real>(numx);
```

Comment:
I moved this out as a tiny optimization. The compiler probably did this already.